frontends/main/src/app-pages/ChannelPage/ChannelPage.test.tsx

```diff
@@ -422,14 +422,6 @@ describe("Channel Pages, Unit only", () => {
         undefined,
       )
     })
-
-    await waitFor(() => {
-      expect(makeRequest).toHaveBeenCalledWith(
-        "get",
-        urls.learningResources.featured({ limit: 12 }),
-        undefined,
-      )
-    })
```

Comment:
This API call was being made before, and is not being made now.

But... I'm not sure why we were testing for it. This was another case of the carousel making an API call while its configuration was loading. (Difference, btw, from the carousel loading its children AFTER the config is supplied.)